### DISCUSSION TITLE
How do you export a file from Toonsquid?

### DISCUSSION THREAD

**Discussion by mcallbee** [keiwando/toonsquid#443] [General]
I have a file with 61 drawings in it. I want to export the entire file with its contents. When I choose the file, the little check mark appears and the options light-up along the bottom...one being "export". When I click on export, nothing happens. What am I doing wrong?

Thanks

**Comment by kavlor** [comment]
Is it a folder your trying to export ?. Folders are just for organising.

**Comment by mcallbee** [comment]
yes. But I was told you cant do that.
